webserver/src/main/java/com/mlaide/webserver/controller/GlobalExceptionHandler.java

```diff
@@ -32,10 +41,51 @@
 
     @ExceptionHandler(value = { NotFoundException.class })
     protected ResponseEntity<Object> handleNotFound(NotFoundException e, WebRequest request) {
-        Error error = new Error(HttpStatus.NOT_FOUND.value(), e.getMessage());
+        Error error = new Error(HttpStatus.NOT_FOUND.value(), HttpStatus.NOT_FOUND.getReasonPhrase());
```

Comment:
Würde `e.getMessage()` lassen. Das ist ein eigener Exception-Typ. In vielen Fällen haben wir eine eigene Exception-Message definiert. In den Fällen in denen wir keine explizite Message haben, ist es gut genug einfach nichts anzugeben. 404 ist ja sprechend genug.

Alternativ könnten wir `e.getMessage()` und als Fallback (falls message null oder empty) dann auf `.getReasonPhrase()` zurückgreifen.

Reply:
2eres lieber.